packages/thirdweb/src/utils/encoding/hex.ts

```diff
@@ -1,52 +1,8 @@
-import { cachedTextDecoder } from "../text-decoder.js";
-import { cachedTextEncoder } from "../text-encoder.js";
-// slightly tweaked re-exports from viem for the moment
-import { assertSize } from "./helpers/assert-size.js";
-import { charCodeToBase16 } from "./helpers/charcode-to-base-16.js";
+import { Hex as ox__Hex } from "ox";
 import type { Hex } from "./helpers/is-hex.js";
```

Comment:
remove this and instead use `Hex.Hex` type from ox?

as in the `Hex` module exports a type `Hex` on it (so `Hex.Hex` to access it)

and that's the correct type

Reply:
Reassigned it to our own type and exported that so it doesn't show as ox's type

Reply:
fair enough for now I think it would be fine to do that though later on